src/Commands/InitCommand.php

```diff
@@ -347,7 +347,7 @@ protected function fillCredentialsAndAccess(string $kebabName): void
                 continue;
             }
 
-            if ($this->confirm("Is {$title}'s admin the same as default one?", true)) {
+            if ($this->adminCredentials && $this->confirm("Is {$title}'s admin the same as default one?", true)) {
```

Comment:
```suggestion
            if (!empty($this->adminCredentials) && $this->confirm("Is {$title}'s admin the same as default one?", true)) {
```

Reply:
@RGO230 please also generate admin for the telescope in case user answer that telescope should have another admin